### DISCUSSION TITLE
Let's get into GH "merge queue" beta?

### DISCUSSION THREAD

**Discussion by pjanotti** [open-telemetry/opentelemetry-dotnet-instrumentation#2188] [Polls]
@open-telemetry/dotnet-instrumentation-approvers and @open-telemetry/dotnet-instrumentation-maintainers 

The GH "merge queue" is in beta, but, it seems a good improvement to our workflow. I would like to give it a try. See https://github.blog/changelog/2023-02-08-pull-request-merge-queue-public-beta/ for more info

**Comment by Kielek** [comment]
I think that we can give it a try.
It looks good in some random repository: https://github.com/t3-oss/create-t3-app/pull/1229
Probably we should update part of our workflows to execute on merge group https://github.com/search?q=%22merge_group%3A%22&type=code

**Reply by pjanotti** [reply]
Let's enable it after 0.6.0 is out.

**Reply by Kielek** [reply]
I have given a try. There is no possibility to report Easy CLA status to queue, so it cannot be merged.
otel-js have had the same issue. They do not found the workaround.

**Reply by Kielek** [reply]
Issue created to support this case: https://github.com/communitybridge/easycla/issues/3846

**Reply by pjanotti** [reply]
SIG: let's revisit and see if we can enable it now.

**Comment by pjanotti** [comment]
SIG: closing at this time since this doesn't seem really needed at this point
